src/pages/CatShow.js

```diff
@@ -1,16 +1,24 @@
 import React, { Component } from "react"
+import { Button } from "reactstrap"
 
 class CatShow extends Component {
   render() {
     console.log(this.props.cat)
     let { cat } = this.props
     return (
-      <>
-        <h1>This is our CatShow!</h1>
-        <p> { cat.name} </p>
-        <p>{ cat.age} </p>
-        <p>{ cat.enjoys} </p>
-      </>
+      <div className="showpage">
+        <h1>Hi! Nice to meet ya!</h1>
+        <p>My names, {cat.name}</p>
+        <p>I'm {cat.age} years old</p>
+        <p>I enjoy {cat.enjoys}</p>
+        <Button href={`/catedit/${this.props.cat.id}`}>
```

Comment:
might need to use `<NavLink>` if you're looking to navigate and style the button

Reply:
Thank you! We will merge and then style and change that if needed